dashboard/src/components/AppView/ServicesTable/ServiceItem.tsx

```diff
@@ -1,32 +1,73 @@
 import * as React from "react";
+import { AlertTriangle } from "react-feather";
 
-import { IResource, IServiceSpec, IServiceStatus } from "../../../shared/types";
+import LoadingSpinner from "../../../components/LoadingSpinner";
+import {
+  IKubeItem,
+  IResource,
+  IResourceRef,
+  IServiceSpec,
+  IServiceStatus,
+} from "../../../shared/types";
 
 interface IServiceItemProps {
-  service: IResource;
+  serviceRef: IResourceRef;
```

Comment:
Maybe serviceRef is redundant? Couldn't you get the name from `service: IKubeItem<IResource>;`?

Reply:
`service` may be still empty, that's why serviceRef may be needed